listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -1,3 +1,39 @@
+# This script is responsible to process user listening history and create and save dataframes to be used
```

Comment:
Maybe it's a good idea to follow the standard docstring format here and near the top, list the inputs that it needs and the outputs it generates. the situation, task, action, result framework is helpful when writing docs explaining large scripts. https://en.wikipedia.org/wiki/Situation,_task,_action,_result